README.md

```diff
@@ -124,32 +132,26 @@ label column.
 
 ## Dataset format
 
-Cleanlab currently only supports text and tabular dataset modalities.
+Cleanlab currently only supports text, tabular, and image dataset modalities.
 (If your dataset contains both text and tabular data, treat it as tabular.)
-The accepted dataset file types are: `.csv`, `.json`, and `.xls/.xlsx`.
 
-Below are some examples of how to format your dataset depending on modality and file type.
+The accepted dataset file types are: `.csv`, `.json`, and `.xls/.xlsx`.
 
-Every dataset must have an **ID column** (i.e. a column containing identifiers that uniquely identify each row) and a
-**label column** (for the prediction task).
+Check below for instructions of formatting different dataset modalities.
 
-Apart from the reserved column name: `clean_label`, You are free to name the columns in your dataset in any way you
-want.
+#### Tabular dataset format
+- dataset must have an **ID column** (`flower_id` in the example below) - a column containing identifiers that uniquely identify each row.
+- dataset must have a **label column** (`species` in the example below) for the prediction task.
+- Apart from the reserved column name: `clean_label`, you are free to name the columns in your dataset in any way you want.
```

Comment:
Clarify that 

There can be some subset of the columns used as features to predict the label, based upon which Cleanlab Studio identifies label issues, and other columns with extra miscellaneous metadata that will be ignored when modeling the labels.